src/Containers/SlidingOutPanels.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import { MainInterface, ResponsiveInterface } from '../Utils/BaseStyles';
 
 interface ImageProps {
-    key: number;
+    id: string;
     imageLink: string;
```

Comment:
Link != Source btw. imageSource would be more appropriate since Link makes me think when I click on the Image it takes me to X.

Reply:
change implemented, link updated to imageSource